code/modules/mob/living/carbon/alien/alien_defense.dm

```diff
@@ -33,13 +33,32 @@ In all, this is a lot like the monkey code. /N
 				add_attack_logs(M, src, "Alien attack", ATKLOG_ALL)
 			else
 				to_chat(M, "<span class='warning'>[name] is too injured for that.</span>")
+	if(islarva(M))
```

Comment:
Also is this still required even if you have the actual spell available now?
Edit, because of the larva not having hands right? Making the transfer spell into a click spell would remove the need for this code

Reply:
UPDATE: transfer plasma from larva aliens is being removed

Reply:
Has been removed